runelite-client/src/main/java/net/runelite/client/plugins/raids/bats/Chest.java

```diff
@@ -0,0 +1,32 @@
+package net.runelite.client.plugins.raids.bats;
+
+import java.awt.Color;
+import lombok.Getter;
+import lombok.Setter;
+
+public class Chest
```

Comment:
Is there a reason why the chest location isn't being stored here? It looks like it would simplify things a bit.

Reply:
It didn't seem necessary.